test/mocha.opts

```diff
@@ -0,0 +1,3 @@
+--require hard-rejection/register
+--require source-map-support/register
+--timeout 10000
```

Comment:
are you sure that mocha lets you pass the require flag twice like this?  I honestly don't know.

Reply:
Yeah, that works. The `--require` option is a pass-through to node.